examples/common/entry/entry_sdl.cpp

```diff
@@ -830,7 +825,7 @@ namespace entry
 							{
 							case SDL_USER_WINDOW_CREATE:
 								{
-									WindowHandle handle = getWindowHandle(uev);
+									WindowHandle handle = reinterpret_cast<WindowHandle>(_uev.data1);
```

Comment:
I prefer you leave getWindowHandle instead this `reinterpret_cast`. Basically ideally no `reinterpret_cast` anywhere.

Reply:
Sure thing!